### DISCUSSION TITLE
support offline maps

### DISCUSSION THREAD

**Discussion by nutpantz** [Hamza417/Positional#127] [Ideas]
i would like to see offline map support. 
mapforge is the usual, or osmamd maps .
many app developer even build their own maps compatible to mapforge
but put in features that they prefer ( like minimal maps but with POIs or elevation data they feel is important)
and strip out other data ( ie a off road maps that had a LOT of detail for bike, hike dirt trails and roads and only major roads and no side streets).

**Comment by Hamza417** [comment]
There was an [OSM build](https://github.com/Hamza417/Positional/releases/tag/Build125) which later got scraped because their Maps SDK was buggy and unstable. There was also a bigger issue about my lack of understanding because Positional was my first app and I just started learning, there wasn't a lot I could do at that time. If it gets better support from the community then I may try to rewrite the whole app adding better features this time around.
